bin.js

```diff
@@ -38,17 +40,44 @@ const cli = meow(`
     },
     lazy: {
       type: 'boolean'
+    },
+    force: {
+      type: 'boolean',
+      default: false
     }
   }
 })
 
+async function addCheckSizes (ipfs, input) {
+  const stats = await Promise.all(input.map(async domain => {
+    const cid = await ipfs.resolve(`/ipns/${domain}`)
+    return ipfs.files.stat(cid)
+  }))
+
+  const size = stats.reduce((acc, v) => acc + v.cumulativeSize, 0)
+
+  if (size >= 10) {
```

Comment:
I believe the `CumulativeSize` is in bytes, and the prompt should be shown for much lower size than 1GB.
I'd go with 10MB (10485760 bytes).
(remember to update comment in line below)

Reply:
@lidel ooops. I know, I was trying to check if the prompt worked and left it that way...